### PR TITLE
Fix "disabled" documents on unread messages

### PR DESCRIPTION
**Pull request by trev-dev**
Jira: EDX-1099
Cc: @SoLetsDev

### CODE REVIEW

File: frontend/src/components/secure-message/MessageDisplay.vue [hunk 1]
```diff
@@ -795,20 +832,25 @@ export default {
...
           this.getExchange();
           this.setSuccessAlert('Success! The note has been removed.');
           this.closeNoteIndex();
         })
         .catch(error => {
           console.error(error);
-          this.setFailureAlert(error?.response?.data?.message ? error?.response?.data?.message : 'An error occurred while trying to remove the note from the Secure Exchange. Please try again later.');
+          let fallback =  'An error occurred while trying to remove the note from the Secure Exchange.' +
+            ' Please try again later.';
+          this.setFailureAlert(error?.response?.data?.message || fallback);
```
Comment (SoLetsDev) [inline frontend/src/components/secure-message/MessageDisplay.vue:853]: whoops we missed this 1/2

File: frontend/src/components/secure-message/MessageDisplay.vue [hunk 2]
```diff
@@ -848,7 +892,9 @@ export default {
         })
         .catch(error => {
           console.error(error);
-          this.setFailureAlert(error?.response?.data?.message ? error?.response?.data?.message : 'An error occurred while adding the note to the Secure Exchange. Please try again later.');
+          let fallback =  'An error occurred while adding the note to the Secure Exchange.' +
+            ' Please try again later.';
+          this.setFailureAlert(error?.response?.data?.message || fallback);
```
Comment (SoLetsDev) [inline frontend/src/components/secure-message/MessageDisplay.vue:897]: whoops we missed this 2/2

File: frontend/src/components/secure-message/MessageDisplay.vue (hunk 1 above)
Comment (trev-dev) [reply frontend/src/components/secure-message/MessageDisplay.vue:853]: Got it!

File: frontend/src/components/secure-message/MessageDisplay.vue (hunk 2 above)
Comment (trev-dev) [reply frontend/src/components/secure-message/MessageDisplay.vue:897]: 🎯

**Review by SoLetsDev — APPROVED** [review]

**Pull request merged**